README.md

```diff
@@ -0,0 +1,35 @@
+# Spots
+
```

Comment:
* Please, make the **modal** match the `mobile` design on 320px https://snipboard.io/Z0TsRH.jpg . It should be **wider** than the cards, but it should not stick to the sides of the screen